test/infrastructure/vm-operator/vm-operator-env-var-patch.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
```

Comment:
Let's add a comment how we got to this file (where we got the value from, maybe with a git tag or somethign if we took it from a specific version of vm-operator)

Reply:
Added